README.md

```diff
@@ -12,19 +12,17 @@ Tenemos diferentes desafíos con un foco específico para cada área de actuaci
 - [Desafio front-end](challenges/frontend.md)
 
 ## Plazo y formato para entrega de la prueba
-Para entregar la prueba usted tiene hasta siete días, contados a partir de la fecha en que usted recibió el desafío.
+Para entregar la prueba usted tiene hasta siete días, contados a partir de la fecha en que recibió el desafío.
 
 Para la entrega, el código debe estar en un repositorio _PRIVADO_ en Github.  
 Para facilitar la identificación del proyecto, por favor nombre el proyecto como `{su-usuario-github}-aptuno-{challengeType}` (e.g `diana2020-aptuno-backend`). Por favor antes de enviar el código valide los permisos de su repositorio. No vamos a revisar repositorios públicos.
 Pedimos por favor que usted cree para su repositorio un **README** con los siguientes items:
 
 - ¿Cómo correr su código localmente?
-- ¿Cómo correr sus pruebas?
-- Agregue un resumen de las decisiones tecnicas (Arquitecturas, Patrones de diseño) tomadas para abordar la prueba.
+- Idealmente un resumen de las decisiones tecnicas (arquitectura, herramientas, patrones de diseño, etc) tomadas para abordar la prueba.
 
-Nosotros valoramos mucho la documentación de los proyectos en ese formato, utilice ese **README** como su carta de presentación.
 
->**Nota**: Para el desafío [backend](challenges/backend.md), le agradecemos proveer un ambiente dockerizado utilizando [docker-compose](https://docs.docker.com/compose/), y así evitarnos la instalación y configuración de herramientas específicas de su entregable.
+>**Opcional**: Para el desafío [backend](challenges/backend.md), le agradecemos proveer un ambiente dockerizado utilizando [docker-compose](https://docs.docker.com/compose/), y así evitarnos la instalación y configuración de herramientas específicas de su entregable.
```

Comment:
Pense que íbamos a eliminar esta por completo

Reply:
es posible eliminarlo, no lo hice pensando en que el experimento de las revisiones "superficiales" no funcione y nos to que volver a ser enfáticos en este item, lo dejé opcional, esperemos como nos va a vemos si lo removemos del todo